jobs/pgbackup/templates/bin/pgbackup

```diff
@@ -62,13 +118,17 @@ backup_databases() {
 	else
 		echo "FAILED to back up!"
 	fi
+	rm -f "/tmp/backup.lock"
 }
 
 echo "pgbackup starting up"
-# FIXME: acquire a lock to ensure we aren't running simultaneous dumps
 
-backup_databases
+# if a backup.lock file exist then we assume that an other dump is running, so we will retry to backup in the next run
+if [ ! -f "/tmp/backup.lock" ]; then
```

Comment:
should probably stop waiting on the lock after a max time, such as 1 day, and clear the backup lock ?